app/channel-settings/client/startup/messageTypes.js

```diff
@@ -29,6 +29,18 @@ Meteor.startup(function() {
 		},
 	});
 
+	MessageTypes.registerType({
+		id: 'room_changed_avatar',
```

Comment:
should we start using semantically correct types or keep old schema? 🤔 
```suggestion
		id: 'room_avatar_changed',
```